README.md

```diff
@@ -1,13 +1,11 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this week's project we used our new skills in Express and Mongodb to build an API which includes POST-request endpoints to create data, and GET-request endpoints to return data. We are building the API to use it on our Happy Thoughts project we did a couple of week ago.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Main problem was to deploy the project. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://ulysses-happy-thoughts-api.herokuapp.com/
```

Comment:
I think you need to provide link to Netlify. Your API will be on Heroku, but Front-end part from the first Happy Thoughts project will be on Netlify. On that front-end part you need only to change URL to your own API.